dp/core/syscall.c

```diff
@@ -199,7 +199,8 @@ static int sys_bpoll(struct bsys_desc __user *d, unsigned int nr)
 				KSTATS_PUSH(idle, NULL);
 
 				start = rdtsc();
-				eth_rx_idle_wait(deadline);
+				if (percpu_get(eth_num_queues) > 0)
```

Comment:
We can get the eth devices using that global variable:
https://github.com/ix-project/ix/blob/master/inc/ix/ethdev.h#L1041

But is it possible and how to know the index of the eth device within the syscall routine is what I couldn't figure out yet.